File: app/src/main/java/ru/roscha_akademii/medialib/main/presenter/MainPresenter.java
package ru.roscha_akademii.medialib.main.presenter;

import com.hannesdorfmann.mosby.mvp.MvpPresenter;

import ru.roscha_akademii.medialib.main.view.MainView;

public interface MainPresenter extends MvpPresenter<MainView> {
    void wannaUpdateVideoList();

    void start();
}

File: app/src/main/java/ru/roscha_akademii/medialib/main/presenter/MainPresenter.kt
package ru.roscha_akademii.medialib.main.presenter

import com.hannesdorfmann.mosby.mvp.MvpPresenter

import ru.roscha_akademii.medialib.main.view.MainView

interface MainPresenter : MvpPresenter<MainView> {
    fun wannaUpdateVideoList()

    fun start()

    fun wannaOpenVideo(id: Long)
}
